tox.ini

```diff
@@ -0,0 +1,123 @@
+[tox]
+envlist = py{36,37}
+
+[testenv]
+skipsdist = true
+usedevelop = true
+passenv = SUNDIALS_INST
+setenv =
+       SUNDIALS_INST =	{env:SUNDIALS_INST:{homedir}/.local}
+       LD_LIBRARY_PATH = {homedir}/.local/lib{:}{env:LD_LIBRARY_PATH:}
+deps =
+     cmake
+     scikits.odes
+     {[testenv:flake8]deps}
+     {[testenv:doctests]deps}
+commands =
+	 python run-tests.py --quick
+
+[testenv:unit]
+commands = python run-tests.py --unit --folder all
+
+[testenv:sundials]
+skip_install = true
+passenv = HOME
+whitelist_externals = git
+deps =
+     wget
+     cmake
+commands =
+         python {toxinidir}/scripts/install_KLU_Sundials.py
+	 - git clone https://github.com/pybind/pybind11.git {toxinidir}/pybind11
```

Comment:
shouldn't this install `scikits.odes` too? Or is it assumed that user will always run sundials then dev? What happens if other way round?

Reply:
Yes, it's assumed that `tox -e sundials` is executed before. It's to avoid going through the sundials build each time one wants to create a virtualenv...
We could proabably merge `tox -e sundials` into `tox -e dev`, but I should check that the sundials script is smart enough to detect a previous installation and not re-download everything! (I don't think it is atm)